plugins/corsair/fu-corsair-bp.c

```diff
@@ -121,6 +124,36 @@ fu_corsair_bp_command(FuCorsairBp *self,
 	return TRUE;
 }
 
+/**
+ * @brief Flush all input reports if there are any.
+ * @self: a #FuCorsairBp
+ *
+ * This function clears any dangling IN reports that
+ * the device may have sent after the enumeration.
+ */
+void
+fu_corsair_bp_flush_input_reports(FuCorsairBp *self)
+{
+	gsize actual_len;
+	g_autofree guint8 *buf = g_malloc0(self->cmd_read_size);
+	g_autoptr(GError) local_error = NULL;
+	GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(self));
+
+	for (guint i = 0; i < CORSAIR_INPUT_FLUSH_ITERATIONS; i++) {
+		if (!g_usb_device_interrupt_transfer(usb_device,
```

Comment:
Just move the error into the loop scope them you dont need to clear it

Reply:
done